.github/workflows/pr-check.yml

```diff
@@ -22,6 +22,10 @@ jobs:
       uses: actions/setup-go@v3
       with:
         go-version: 1.18
+    - name: Set up kubebuilder
+      uses: RyanSiu1995/kubebuilder-action@v1.3.1
+      with:
+        version: 2.3.2
```

Comment:
kubebuilder is now required to run the tests in `api/v1alpha1/webhook_suite_test.go`

Reply:
Could you elaborate on why kubebuilder is needed? I'm not sure I like including an action for this.

Reply:
I think that's because of the following annotation `//+kubebuilder:scaffold:webhook`
Could we simple remove it?

Reply:
I will have to check if `//+kubebuilder:scaffold:webhook` can be removed. The kubebuilder comments were added automatically when I ran `operator-sdk create webhook --group che --version v1alpha1 --kind KubernetesImagePuller  --defaulting --programmatic-validation --force` to generate the webhooks.

@amisevsk I think that can replace it, I will take a closer look

Reply:
You're right @amisevsk it's not needed, new commit fixes the test run failure: https://github.com/che-incubator/kubernetes-image-puller-operator/pull/115/commits/6f70e83ae5db55711b27a9e99538d40c57a27edf